client/structure/profile.js

```diff
@@ -17,6 +18,7 @@ export const ProfileStructure = {
     defaultConfig: {
         profileWidgets: [],
         profileModules: [],
+        profilePhotoLayers: [VerificationBadge],
```

Comment:
Why is this here? Wouldn't we just turn on the verification feature on profiles by default, rather than having this hard-coded into the structure?

We do need to add profilePhotoLayers to the structure, so it's accessible as a config slot, but I think it should default to empty.